add-tourpackage.php

```diff
@@ -0,0 +1,483 @@
+<?php require 'connection.php'; ?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Package</title>
+    <link rel="stylesheet" href="bootstrap.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-12 col-md-8 col-lg-8 offset-md-2 offset-lg-2">
+                <div class="row">
+                    <div class="col-12">
+                        <center>
+                            <h4>Add New Tour Package</h4>
+                        </center>
+                    </div>
+                    <div class="col-12">
+                        <hr>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Name of the Package</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="name">
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-12 mt-3">
+                        <div class="row">
+                            <div class="col-12 col-md-4 col-lg-4">
+                                <label>Price</label>
+                            </div>
+                            <div class="col-12 col-md-8 col-lg-8">
+                                <input type="text" class="w-100" id="price">
```

Comment:
If you have to use an input element for `price`. You can use an attribute called `pattern`. It helps to prevent invalid value entered by the user.

Refer [this](https://stackoverflow.com/questions/29603080/html5-input-type-number-with-regex-pattern)

Reply:
Got it